bi_sql_editor/models/bi_sql_view_field.py

```diff
@@ -86,6 +86,12 @@ class BiSQLViewField(models.Model):
 
     index_name = fields.Char(compute="_compute_index_name")
 
+    is_many2one_clickable = fields.Boolean(
+        default=True,
+        help="If the field is of type Many2one and the option is enabled, the"
+        "field will be clickable on the list view.",
```

Comment:
```suggestion
        " field will be clickable on the list view.",
```
I can see a space is missing